packages/formdescriptioneditors/frontend/sirius-components-formdescriptioneditors/src/SplitButtonWidget.tsx

```diff
@@ -0,0 +1,352 @@
+import { GQLButtonStyle, getTextDecorationLineValue } from '@eclipse-sirius/sirius-components-forms';
+import Button from '@material-ui/core/Button';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import Grow from '@material-ui/core/Grow';
+import MenuItem from '@material-ui/core/MenuItem';
+import MenuList from '@material-ui/core/MenuList';
+import Paper from '@material-ui/core/Paper';
+import Popper from '@material-ui/core/Popper';
+import Typography from '@material-ui/core/Typography';
+import { Theme, makeStyles } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import HelpOutlineOutlined from '@material-ui/icons/HelpOutlineOutlined';
+import React, { useContext, useEffect, useRef, useState } from 'react';
+import { addWidgetMutation } from './FormDescriptionEditorEventFragment';
+import {
+  GQLAddWidgetInput,
+  GQLAddWidgetMutationData,
+  GQLAddWidgetMutationVariables,
+  GQLAddWidgetPayload,
+  GQLErrorPayload,
+} from './FormDescriptionEditorEventFragment.types';
+import { SplitButtonWidgetState } from './SplitButton.type';
+import { SplitButtonWidgetProps } from './WidgetEntry.types';
+
+const isErrorPayload = (payload: GQLAddWidgetPayload): payload is GQLErrorPayload =>
+  payload.__typename === 'ErrorPayload';
+
+const actionStyle = (theme: Theme, style: GQLButtonStyle): React.CSSProperties => {
+  const actionStyle: React.CSSProperties = {
+    backgroundColor: style.backgroundColor ? getCSSColor(style.backgroundColor, theme) : theme.palette.primary.light,
+    color: style.foregroundColor ? getCSSColor(style.foregroundColor, theme) : 'white',
+    fontSize: style.fontSize ? style.fontSize : undefined,
+    fontStyle: style.italic ? 'italic' : undefined,
+    fontWeight: style.bold ? 'bold' : undefined,
+    textDecorationLine:
+      style.underline || style.strikeThrough
+        ? getTextDecorationLineValue(style.underline, style.strikeThrough)
+        : undefined,
+  };
+
+  return actionStyle;
+};
+
+const imageStyle = (theme: Theme, iconOnly: boolean): React.CSSProperties => {
+  return {
+    marginRight: iconOnly ? theme.spacing(2) : theme.spacing(0),
+    width: theme.spacing(2),
+    height: theme.spacing(2),
+  };
+};
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  bottomDropArea: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+    backgroundColor: 'whitesmoke',
+    borderRadius: '10px',
+    color: 'gray',
+    height: '30px',
+    width: '100%',
+  },
+  helpOutlineOutlined: {
+    marginLeft: theme.spacing(1),
+    fontSize: theme.spacing(2),
+  },
+  dragOver: {
+    borderWidth: '1px',
+    borderStyle: 'dashed',
+    borderColor: theme.palette.primary.main,
+  },
+  selected: {
+    color: theme.palette.primary.main,
+  },
+  containerAndLabel: {
+    margin: 0,
+    padding: 0,
+    borderWidth: 1,
```

Comment:
Could you change it to `'1px'` please? Same for the margin and padding above, and borderRadius below.